phpstan.neon

```diff
@@ -1,3 +1,10 @@
 parameters:
     level: 8
     paths: [ src ]
+    ignoreErrors:
+        - '#Comparison operation ">" between int and RdKafka\\TopicPartition results in an error.#'
+        - '#Call to static method fromSchema\(\) on an unknown class Jobcloud\\Avro\\Validator\\RecordRegistry#'
+        - '#PHPDoc tag @throws with type [a-zA-Z0-9\\_]+#'
+        - '#Parameter \#[0-9] [a-z\$]+ of method [a-zA-Z0-9\\_]+::setOffset\(\) expects string, int given.#'
+        - '#Parameter \#[0-9] [a-z\$]+ of class Jobcloud\\Kafka\\Exception\\AvroValidatorException constructor expects string, string\|false given.#'
```

Comment:
Not sure where this is occuring, but we should be able to fix this, ugliest way to do it would be to use `(string)`

Reply:
It's where json_encode() is used for avro schema and exception